### ISSUE TITLE
loss为nan？请问一般是什么问题呢？

### ISSUE THREAD

**Issue by destiny311** [LeeRel1991/faceAlignment_tensorflow#18]
lodat dataset: afw/trainset
lodat dataset: helen/trainset
lodat dataset: lfpw/trainset
dan [2019-07-06 13:01:04,498] INFO total samples: 31480
dan [2019-07-06 13:01:04,499] INFO config: batch_size: 64
 num_epochs:100
stage: 1

dan [2019-07-06 13:01:19,398] INFO Iter: 50, Lr: 0.000997961033136, Loss: nan, spend: 7.3503s
dan [2019-07-06 13:01:24,461] INFO Iter: 100, Lr: 0.000995926209725, Loss: nan, spend: 4.9818s
dan [2019-07-06 13:01:29,546] INFO Iter: 150, Lr: 0.000993895344436, Loss: nan, spend: 5.0000s
dan [2019-07-06 13:01:34,642] INFO Iter: 200, Lr: 0.000991868786514, Loss: nan, spend: 5.0116s
dan [2019-07-06 13:01:39,725] INFO Iter: 250, Lr: 0.000989846419543, Loss: nan, spend: 5.0066s
dan [2019-07-06 13:01:44,814] INFO Iter: 300, Lr: 0.000987828127109, Loss: nan, spend: 5.0074s
dan [2019-07-06 13:01:49,933] INFO Iter: 350, Lr: 0.00098581390921, Loss: nan, spend: 5.0298s
dan [2019-07-06 13:01:55,036] INFO Iter: 400, Lr: 0.000983803765848, Loss: nan, spend: 5.0121s
dan [2019-07-06 13:02:00,185] INFO Iter: 450, Lr: 0.000981797813438, Loss: nan, spend: 5.0426s
dan [2019-07-06 13:02:05,947] INFO Iter: 500, Lr: 0.000979795819148, Loss: nan, spend: 5.6606s
dan [2019-07-06 13:02:11,034] INFO Iter: 550, Lr: 0.000977798015811, Loss: nan, spend: 5.0075s
dan [2019-07-06 13:02:16,117] INFO Iter: 600, Lr: 0.000975804403424, Loss: nan, spend: 5.0085s
dan [2019-07-06 13:02:21,216] INFO Iter: 650, Lr: 0.000973814632744, Loss: nan, spend: 5.0230s
dan [2019-07-06 13:02:26,313] INFO Iter: 700, Lr: 0.000971828994807, Loss: nan, spend: 5.0201s
dan [2019-07-06 13:02:31,401] INFO Iter: 750, Lr: 0.000969847431406, Loss: nan, spend: 5.0141s
dan [2019-07-06 13:02:36,495] INFO Iter: 800, Lr: 0.000967869884335, Loss: nan, spend: 5.0175s
dan [2019-07-06 13:02:41,664] INFO Iter: 850, Lr: 0.000965896470007, Loss: nan, spend: 5.0589s
dan [2019-07-06 13:02:46,814] INFO Iter: 900, Lr: 0.000963926955592, Loss: nan, spend: 5.0448s
dan [2019-07-06 13:02:51,914] INFO Iter: 950, Lr: 0.000961961515713, Loss: nan, spend: 5.0153s
dan [2019-07-06 13:02:57,003] INFO Iter: 1000, Lr: 0.000960000033956, Loss: nan, spend: 5.0104s
dan [2019-07-06 13:03:02,103] INFO Iter: 1050, Lr: 0.000958042568527, Loss: nan, spend: 5.0215s
dan [2019-07-06 13:03:07,199] INFO Iter: 1100, Lr: 0.000956089061219, Loss: nan, spend: 5.0192s
dan [2019-07-06 13:03:12,306] INFO Iter: 1150, Lr: 0.000954139628448, Loss: nan, spend: 5.0209s
dan [2019-07-06 13:03:17,444] INFO Iter: 1200, Lr: 0.000952194095589, Loss: nan, spend: 5.0420s
dan [2019-07-06 13:03:22,542] INFO Iter: 1250, Lr: 0.000950252520852, Loss: nan, spend: 5.0209s
